app/helpers/search_bar_helper.rb

```diff
@@ -0,0 +1,25 @@
+module SearchBarHelper
+  def mount_search_bar(
+    id:,
+    controller: nil,
+    url: nil,
+    search_query: nil,
+    use_bookmarks: true,
+    use_key_shortcuts: true
+  )
+    bookmarks = {
+      url: api_bookmarks_path,
+      canCreate: authorizer.can?(:create_bookmarks),
+      documentationUrl: documentation_url("4.1.5Searching")
+    }
```

Comment:
if `use_bookmarks` is `false` there's no need to calculate these (specifically the authorization which can be a bit slow), just adding a `if use_bookmarks` at the end of the line

Reply:
Updated it :)